testdata/python-small/docfx.json

```diff
@@ -0,0 +1,26 @@
+{

```

Comment:
license & copyright, possibly embedded in the json?

Reply:
Done.